src/Patches.php

```diff
@@ -351,18 +351,21 @@ protected function getAndApplyPatch(RemoteFilesystem $downloader, $install_path,
       $downloader->copy($hostname, $patch_url, $filename, FALSE);
     }
 
-    // Modified from drush6:make.project.inc
-    $patched = FALSE;
-    // The order here is intentional. p1 is most likely to apply with git apply.
-    // p0 is next likely. p2 is extremely unlikely, but for some special cases,
-    // it might be useful.
-    $patch_levels = array('-p1', '-p0', '-p2');
-    foreach ($patch_levels as $patch_level) {
-      $checked = $this->executeCommand('cd %s && git --git-dir=. apply --check %s %s', $install_path, $patch_level, $filename);
-      if ($checked) {
-        // Apply the first successful style.
-        $patched = $this->executeCommand('cd %s && git --git-dir=. apply %s %s', $install_path, $patch_level, $filename);
-        break;
+    // Check if code is patched already.
+    $patched = $this->executeCommand('cd %s && git --git-dir=. apply -R --check -p1 %s', $install_path, $filename);
```

Comment:
Honestly I dont know the true meaning of patch level. But at that time it seemed important to me not to mess up a reverse check at higher level. And thats the only way I got it to work on my setup. In combination with the shared-packages-plugin.